monitor-payments/index.js

```diff
@@ -0,0 +1,44 @@
+const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+// client for indexerV2
+const algosdk = require('algosdk');
+const indexer_token = "";
+const indexer_server = "http://localhost";
+const indexer_port = 8980;
+const indexerClient = new algosdk.Indexer(indexer_token, indexer_server, indexer_port);
+
+// this is the address we will monitor for new payments received.
```

Comment:
could you please add a link to config here ?

Reply:
i don't follow. config for address `WWYNX3TKQYVEREVSW6QQP3SXSFOCE3SKUSEIVJ7YAGUPEACNI5UGI4DZCE` ?

Reply:
link from where we get this address.